share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -408,6 +428,32 @@ def postJob (rscApi: RemoteCacheApi) (job: Job) (_wakeroot: String) (hidden: Str
         rscApi
         | rscApiPostStringBlob "stderr" stderr
 
+    # Due to a side effect of filtering outputs its possible for a job to create both a file
+    # and a symlink to that file, but the only output said file. This will break the build
+    # and is explicitly disallowed. Panic if it occurs.
+    require True =
+        require False = symlinksUpload.len == 0
+        else True
+
+        # Get the list of potential targets the job may have uploaded
+        require Pass symlinkTargets =
+            symlinksUpload
+            | findFail
+            |< map getCachePostRequestOutputSymlinkPath
+        else True
```

Comment:
If we are unable to get the target for an output symlink, we are assuming that means this job didn't output it?
Or are we just choosing to ignore that error here and let some other part of the code handle that failure.

Reply:
> we just choosing to ignore that error here and let some other part of the code handle that failure

This one. The only failure here is a failure to `readlink` and if that fails then another part of the code (just a few lines below) will fully abort the job push